### PR TITLE
Card workspace

### PR DESCRIPTION
**Pull request by tenzgass**
Missing the "cloudinary photos"

### CODE REVIEW

File: app/assets/stylesheets/components/_card-workspace.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  }
+
+  .card:before {
+    content: '';
+    display: block;
+    padding-bottom: 150%;
+  }
+  
+  .card_background {
+    background-image: url("https://images.unsplash.com/photo-1524758631624-e2822e304c36?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80");
```
Comment (Simon-Busch) [inline app/assets/stylesheets/components/_card-workspace.scss:33]: we said it should be in the html, not the css so we can link to the instance

File: app/assets/stylesheets/config/_colors.scss [hunk 2]
```diff
@@ -12,5 +12,7 @@ $light-gray: #F4F4F4;
 
 // colors
 $light-grey-1: #d0d0d0;
-$dark-grey-1: #b7b7b7;
-$body: #FAF8E7;
\ No newline at end of file
+$primary-color: #F97350;
+$yellow: #FAD06C;
+$light-beige: #F7F2E5;
+$medium-gray: #B7AC9A;
```
Comment (Simon-Busch) [inline app/assets/stylesheets/config/_colors.scss:18]: leave this below : 
$dark-grey-1: #b7b7b7;
$body: #FAF8E7;

**Review by Simon-Busch — CHANGES_REQUESTED** [review]
Please check my comments

**Review by Simon-Busch — CHANGES_REQUESTED** [review]
Shouldn't  merge this branch, it's the same as Font #19

**Comment by ericblandet** [comment]
doublon

**Pull request closed without merging**